webpack.config.js

```diff
@@ -24,12 +25,14 @@ module.exports = {
       process: require.resolve('process/browser'),
       Buffer: ['buffer', 'Buffer'],
     }),
+    new ForkTsCheckerWebpackPlugin(),
```

Comment:
we are using tsc to generate builds so this won't be affective in catching errors like the one in the issue